src/openrct2/title/TitleSequenceManager.cpp

```diff
@@ -285,7 +285,7 @@ namespace TitleSequenceManager
 
     static void GetUserSequencesPath(utf8* buffer, size_t bufferSize)
     {
-        platform_get_user_directory(buffer, "title sequences", bufferSize);
+        platform_get_user_directory(buffer, "sequence", bufferSize);
```

Comment:
This function should not be needed any more, and code that calls this function should use `PlatformEnvironment::DirectoryNamesOpenRCT2` instead.